<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Row</class>
 <widget class="QWidget" name="Row">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MyRow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>791</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QPushButton" name="DeleteBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Topic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListView" name="listMails">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RepeteBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Повтор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
